link/src/main/java/com/stripe/android/link/ui/signup/SignUpViewModel.kt

```diff
@@ -43,17 +42,17 @@ internal class SignUpViewModel @Inject constructor(
         if (linkAccountManager.hasUserLoggedOut(customerEmail)) null else customerEmail
 
     val merchantName: String = args.merchantName
-    val emailElement: SectionFieldElement =
-        EmailSpec.transform(mapOf(IdentifierSpec.Email to prefilledEmail))
+    val emailController: TextFieldController = SimpleTextFieldController
+        .createEmailSectionController(prefilledEmail)
 
     /**
      * Emits the email entered in the form if valid, null otherwise.
      */
     private val consumerEmail: StateFlow<String?> =
-        emailElement.getFormFieldValueFlow().map { formFieldsList ->
+        emailController.formFieldValue.map {
             // formFieldsList contains only one element, for the email. Take the second value of
             // the pair, which is the FormFieldEntry containing the value entered by the user.
```

Comment:
Same, comments are not valid anymore

Reply:
I missed these comments, but I will fix it in my next PR, sorry about that.

Reply:
resovled in this PR: https://github.com/stripe/stripe-android/pull/4992